docs/docs/administration/install.md

```diff
@@ -131,6 +131,25 @@ $ grep -c ^processor /proc/cpuinfo
 8
 ```
 
+Next, if you choose the option -e for Let's Encrypt, check that your server has the port 80 and 443 open

```

Comment:
```suggestion
Next check that your server has the port 80 and 443 open

```

dropping the mention of Let's Encrypt, otherwise all looks good, thank you!